src/routes/intro/SlideFooter.js

```diff
@@ -105,18 +93,55 @@ class SlideFooter extends React.Component<PropsType> {
     </DotsContainer>
   }
 
-  render () {
-    const { currentSlide, slideCount, theme } = this.props
+  renderStandardFooter = (): React.Node => {
```

Comment:
Can be extracted into separate components like: 'StandartFooter', 'CustomizableSettingsFooter'

Reply:
imo it's fine like this

Reply:
Imo its fine like it is